data/presets/building/roof.json

```
@@ -8,7 +8,8 @@
         "area"
     ],
     "tags": {
-        "building": "roof"
+        "building": "roof",
+        "layer": 1
     },
```

I think this will fix @tyrasd comment:
https://github.com/openstreetmap/id-tagging-schema/pull/227#pullrequestreview-830575632

Keen to get this in too having hit the same issue:
```suggestion
        "building": "roof"
    },
    "addTags": {
        "building": "roof",
        "layer": "1"
    },
```

---

thanks!